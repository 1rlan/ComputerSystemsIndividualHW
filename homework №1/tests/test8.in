20 978165790
1412656873 113780956 978165790 708314342 2081498829 978165790 978165790 1009595470 978165790 978165790 1671950930 978165790 1687994207 338142310 2043925189 978165790 607616260 636241872 682048642 978165790 
