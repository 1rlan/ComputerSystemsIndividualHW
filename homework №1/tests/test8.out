1412656873 113780956 708314342 2081498829 1009595470 1671950930 1687994207 338142310 2043925189 607616260 636241872 682048642
